packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.css

```diff
@@ -1,6 +1,7 @@
 .generated-answer__card-header {
   justify-content: space-between;
   align-items: center;
+  margin-top: 0.6rem;
```

Comment:
This issue is only happening with the hosted search page and only when the integrated search box is not being used, so this is not the problem of this component internally.

Thus, to add this space, it's better to add it in the container around your component.
and it's better to use SLDS CSS classes.

This would be something doable in the C4SF package,  in the component that displays the HSP, and when the  integrated search box is not used, you can don this:

```
<div class="slds-m-top_x-small">
 RGA Component...
</div>
```

Reply:
ok I will create a seperate PR in C4SF package for that then!